### DISCUSSION TITLE
academic citation?

### DISCUSSION THREAD

**Discussion by myeomans** [microsoft/presidio#879] [Q&A] [answered]
Cool software! Is there a preferred way to cite this package if we use it in an academic paper? I couldn't find anything in the documentation. I'm thinking about the text anonymization tool, specifically, but I suppose this could be answered more broadly

**Comment by omri374** [comment, marked as answer]
Hi @myeomans! Thanks for asking. Most citations use the repo's url, but this would be a more formal citation:

```
@misc{MsPresidio,
     AUTHOR = {Mendels, Omri and Peled, Coby and Vaisman Levy, Nava and Hart, Sharon and Rosenthal, Tomer and Lahiani, Limor and others},
     TITLE = {{Microsoft Presidio}: Context aware, pluggable and customizable PII anonymization service for text and images},
     YEAR = {2018},
     ORGANIZATION = {Microsoft},
     URL = {[https://microsoft.github.io/presidio}](https://microsoft.github.io/presidio%7D)
}
```
